README.md

```diff
@@ -1,28 +1,51 @@
-# Project Name
-Put here a short paragraph describing your project. 
-Adding an screenshot or a mockup of your application in action would be nice.  
+# Library Room Auto-Scheduler
+For Temple students who need to schedule rooms in Charles Library farther in advance, the Library Room Auto-Scheduler is a room booking app that allows you to automatically schedule rooms as soon as they’re available. Unlike Charles’s own scheduler, our product allows proactive students to book rooms more than 2-3 days in advance without needing to navigate through the library’s appointment interface.
+
+![This is a screenshot.](images/login.png)
+![This is a screenshot.](images/booking.png)
 
-![This is a screenshot.](images.png)
 # How to run
-Provide here instructions on how to use your application.   
-- Download the latest binary from the Release section on the right on GitHub.  
-- On the command line uncompress using
-```
-tar -xzf  
-```
-- On the command line run with
+- For MacOS users : Download the latest executable from the Release section on the right on GitHub. 
+
 ```
-./hello
 ```
-- You will see Hello World! on your terminal. 
+- A login window should pop up.
 
 # How to contribute
-Follow this project board to know the latest status of the project: [http://...]([http://...])  
+Follow this project board to know the latest status of the project: [http://...]([https://github.com/cis3296s22/libraryroomscheduler/projects/2])  
 
 ### How to build
-- Use this github repository: ... 
-- Specify what branch to use for a more stable release or for cutting edge development.  
-- Use InteliJ 11
-- Specify additional library to download if needed 
-- What file and target to compile and run. 
-- What is expected to happen when the app start. 
+- Use this github repository
+- Use Visual Studio Code
+- Install:
+
+```
+pip3 install selenium
+pip3 install kivy
+pip3 install webdriver_manager
+pip3 install pyinstaller
+
+```
```

Comment:
we may want to think about packaging some sort of virtual environment with these (and python) locked to a specific version. I'll start checking that out